internal/testhelpers/make.go

```diff
@@ -12,10 +12,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const (
```

Comment:
removed so that we stop hard-coding this and instead use the test case name as the task queue for each test case.